services-custom/dynamodb-enhanced/src/main/java/software/amazon/awssdk/enhanced/dynamodb/internal/converter/attribute/MapAttributeConverter.java

```diff
@@ -47,25 +47,28 @@
  *
  * <p>
  * A builder is exposed to allow defining how the map, key and value types are created and converted:
+ * <p>
  * <code>
- * AttributeConverter<Map<MonthDay, String>> mapConverter =
+ * {@literal AttributeConverter<Map<MonthDay, String>> mapConverter =
  * MapAttributeConverter.builder(EnhancedType.mapOf(Integer.class, String.class))
  * .mapConstructor(HashMap::new)
  * .keyConverter(MonthDayStringConverter.create())
  * .valueConverter(StringAttributeConverter.create())
- * .build();
+ * .build();}
```

Comment:
Why @literal and not @code?

Reply:
The ```@literal``` is for the ```'<'``` and``` '>'``` signs, they are special signs in html, thus they cannot be recognize as the other signs, adding ```@literal``` guarantees that all the strings contained are just what they should be looked like. Usually this annotation is for special signs i.e ```'<', '>', '@'```